.env.example

```
@@ -62,6 +62,7 @@ CHATBOT_INTERNAL_URL=http://chatbot:5007
 FLASK_SECRET_KEY=
 FLASK_PORT=5080
 AUTH_SECRET=
+INTERNAL_API_SECRET=  # Auto-generated by 'make init'. Required for production.
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)